asm/include/field_take_photo.inc

```diff
@@ -53,16 +53,16 @@
 .public sub_0205C78C
 .public sub_0205C790
 .public PlayerAvatar_GetSpriteByStateAndGender
-.public CreateSpecialFieldObject
+.public MapObject_Create
 .public sub_0205E38C
 .public MapObject_Remove
 .public sub_0205E420
-.public sub_0205EEF4
+.public MapObjectManager_GetNextObjectWithFlagFromIndex
 .public sub_0205F1A0
 .public MapObject_SetFlagsBits
 .public MapObject_ClearFlagsBits
 .public MapObject_SetID
-.public MapObject_GetGfxID
+.public MapObject_GetSpriteID
```

Comment:
aside: pokeplat usually drops the 'Get' terminology in accessor functions, thoughts on standardizing that? it's readable and clear to me either way but we should prolly standardize it sooner rather than later

Reply:
imo we really need to have get, esp when we have a set of the same thing
`MapObject_SetSpriteID` and `MapObject_SpriteID` doesn't really make sense